### ISSUE TITLE
Failing to load scratch-trained weights

### ISSUE THREAD

**Issue by jazzlover06** [matterport/Mask_RCNN#1141]
I have a .h5 file of scratched-trained weights from a previous use case that I would like to leverage within the Mask RCNN framework. They were created using the ResNet50 Architecture as downloaded from Keras, so ought to fit into the ResNet50 Backbone. Indeed, the model compiles and looks like it is fine-tuning, but the rpn_bbox_loss remains "nan" and tensorboard confirms no real training is happening. I have tried greatly reducing the learning rate to encourage 'warm up', but this does not remedy the problem. 

Even having popped the classification dense layer from my scratch-trained weights, they do seem subtly different from the Keras 'no-top' download within model.py . Is it possible to define a custom backbone architecture that fits my weights? Are there really different architectures within ResNet50? 

Many thanks for any insight.
